.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -31,7 +31,7 @@ body:
     label: Issue Type
     # FIXME: Once GitHub allows defining the default choice, update this
     options:
-    - Bug Report
+      - Bug Report
```

Comment:
The top-level list (under `body:`) is unindented. Do you really want to do this in selected places? It's probably worth using the same style across the document.